README.md

```diff
@@ -73,6 +73,20 @@ class date(object):
     def weekday(self) -> int: ...
 ```
 
+## Conventions
+
+* At the time of this writing, `unicode` arguments, in Python 2 stubs, are
+  interpreted by type-checkers as `Union[bytes, unicode]` (so it means the same
+  as `Text`).
+  Even so, in Python 2, whenever possible, use `unicode` if that's the only
+  possible type, and `Text` if it can be either `unicode` or `bytes`.
+* Most type-checkers interpret optional parameters of the form `x : Foo = None`
+  as `x : Optional[Foo] = ...`. (So the former is a shortcut for the latter)
+  However, the more explicit latter form is preferred, for new contributions.
```

Comment:
I think Guido actually changed this throughout typeshed already, so we should just enforce it for all code.